staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/controller_ref.go

```diff
@@ -52,14 +56,12 @@ func GetControllerOfNoCopy(controllee Object) *OwnerReference {
 
 // NewControllerRef creates an OwnerReference pointing to the given owner.
 func NewControllerRef(owner Object, gvk schema.GroupVersionKind) *OwnerReference {
-	blockOwnerDeletion := true
-	isController := true
 	return &OwnerReference{
 		APIVersion:         gvk.GroupVersion().String(),
 		Kind:               gvk.Kind,
 		Name:               owner.GetName(),
 		UID:                owner.GetUID(),
-		BlockOwnerDeletion: &blockOwnerDeletion,
-		Controller:         &isController,
+		BlockOwnerDeletion: newBooleanRef(true),
+		Controller:         newBooleanRef(true),
```

Comment:
there's a `ptr.To` pkg https://github.com/kubernetes/utils/blob/master/pointer/pointer.go

Reply:
@dprotaso Yeah! I already added this package after getting your comment.

Reply:
@dprotaso after adding the following package one of the test case fails. will that be a problem? if so, can you tell me how to get rid of this?